samples/getting_started/RandomBitsNISQ.qs

```diff
@@ -0,0 +1,31 @@
+/// # Summary
+/// Simple Quantum Random Number Generator sample
+///
+/// # Description
+/// This program implements a quantum random number generator by setting qubits
+/// in superposition and then using the measurement results as random bits.
+/// This is equivalent to generating a random number in the range of 0..2ᴺ-1.
+operation Main() : Result[] {
```

Comment:
I don't think this file needs the "NISQ" suffix. We generally only use that to differentiate between two versions of the same sample/algorithm when one can run in Base profile and the other cannot. There isn't a non-NISQ RandomBits.qs to differentiate this one from, so this one can use that name.

Reply:
Agreed. This was a result of renaming QRNGNISQ (yuck!) and now it doesn't make sense to have NISQ any longer. Renamed.